### PR TITLE
Activate only office

### PR DESCRIPTION
**Pull request by Edmondi-Kacaj**
Hello @mirjan-hoffmann 

since we move everything in docker, I think is better to have also the onlyoffice in the docker version (btw: it has some problem in installing the old one).

This pull request is about installing and running onlyoffice in edu-sharing-box

### CODE REVIEW

File: ansible/roles/onlyoffice/templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    container_name: onlyoffice-postgresql
+    image: postgres:{{onlyoffice_database_postgress_version | default("latest",true)}}
+    environment:
+      - POSTGRES_DB={{onlyoffice_database.db_name | default("onlyoffice",true)}}
+      - POSTGRES_USER={{onlyoffice_database.db_user | default("onlyoffice",true)}}
+      - POSTGRES_PASSWORD={{onlyoffice_database.db_password | default("onlyoffice",true)}}
+      - PGDATA= {{onlyoffice_database.db_data | default("/data/postgres",true)}}
+    restart: unless-stopped
+    expose:
+      - "127.0.0.1:{{ onlyoffice_database.db_port| default('5432',true)}}:5432"
```
Comment (mirjan-hoffmann) [inline ansible/roles/onlyoffice/templates/docker-compose.yml.j2:60]: Using `5432` as default is likely to cause a conflict with the edu-sharing-psql port, isn't it?

File: ansible/group_vars/all.yml [hunk 2]
```diff
@@ -2,13 +2,16 @@
 edu_sharing_host: 192.168.98.101
 edu_sharing_domain: '{{edu_sharing_host}}'
 edu_sharing_url: '{{ edu_sharing_protocol | default("http") }}://{{edu_sharing_host}}/edu-sharing'
-edu_version: 6.0.1
+edu_version: 8.0.0
```
Comment (mirjan-hoffmann) [inline ansible/group_vars/all.yml:5]: This doesn't / shouldn't belong in this PR, but for my sake we can update it

**Review by mirjan-hoffmann — CHANGES_REQUESTED** [review]

**Comment by mirjan-hoffmann** [comment]
@Edmondi-Kacaj Can you have a look please?

File: ansible/group_vars/all.yml (hunk 2 above)
Comment (Edmondi-Kacaj) [reply ansible/group_vars/all.yml:5]: Hello @mirjan-hoffmann 

Yes, you are right, even that we already have in prod 8.0.0, we can update it but this is not part of pull request and I will remove

File: ansible/roles/onlyoffice/templates/docker-compose.yml.j2 (hunk 1 above)
Comment (Edmondi-Kacaj) [reply ansible/roles/onlyoffice/templates/docker-compose.yml.j2:60]: With the edu-sharing docker version it will not cause any conflict, but if somehow it causes, the default on `5432`, then we can override the variable even though I haven't included it in the `onlyoffice_database.` object, we can include and override.

What do you think?

Comment (Edmondi-Kacaj) [reply ansible/roles/onlyoffice/templates/docker-compose.yml.j2]: Hello @mirjan-hoffmann 

I change the port to `5433` so I think it will be OK
